builder/builder.go

```diff
@@ -86,6 +86,70 @@ func NewFromConfig(conf string) *Builder {
 	return b
 }
 
+// CreateDefaultConfig creates a default builder.conf using the active
+// directory as base path for the variables values.
+func (b *Builder) CreateDefaultConfig() error {
+	pwd, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+
+	builderconf := filepath.Join(pwd, "builder.conf")
+
+	err = helpers.CopyFileNoOverwrite(builderconf, "/usr/share/defaults/bundle-chroot-builder/builder.conf")
+	if os.IsExist(err) {
+		fmt.Printf("File '%s' already exists. Skipping...\n", builderconf)
+		return nil
+	} else if os.IsNotExist(err) {
+		fmt.Println("Template for 'builder.conf' not found. Please create it manually.")
+		return nil
```

Comment:
Indeed. The reason why it doesn't is because the original patch didn't intent to cause any workflow change in the way mixer-init runs. I'm not sure if it still makes sense anymore since you need a builder.conf to run mixer-init anyway,
Anyway, this should go away once we move to generating the template with the config parser in my next patch.

Reply:
Agree with everyone: this is likely a valid error case that, for now, will likely never happen on a CLR system. 

I think the biggest thing to note is that this is a temporary solution to move toward _generation_ of the build conf. I'm asking for this to go in so I have somewhere to put in modifications to the conf for the work I'm doing. As @rchiossi pointed out, this will be replaced going forward with a more full-fledged "generate the conf from scratch using viper" solution.

Reply:
I'm not the one that will make the decision whether or not to merge this PR, but why not just implement the more `full-fledged "generate the conf from scratch using viper"` solution now? I don't see the benefit of merging temporary code just so you can put in modifications for another feature on top of it. If the conf generation is a prereq of the other feature, just fully implement the other feature first.

Reply:
I share your opinion like I mentioned on my last comment on the original PR (#39).
The reason why I ported this old PR is because Kevin needed the template generation code for his PR and I don't know how long it will that to get the full-fledged one reviewed and integrated.

Reply:
@kevincwells is there a reason it does not make sense to fully implement the prereq for your work before doing your work?

Reply:
For normal situations it does, but this is a very unique case were this prereq has been going for over a month. It just seems reasonable in his case to expect it to be in by now.

Reply:
@matthewrsj -- I guess what @rchiossi said. The main reason is that the code I've been working on is effectively done (#107), and I don't want to have it hanging until a more full-fledged implementation is done, especially as it interfaces with the BCB rewrite (#102). As mentioned, this code has been outstanding in varying states of approval for over a month. The full-fledged implementation is coming as part of a total config-parsing rewrite that integrates the Viper library. It shouldn't take an incredible amount of time, but it's also non-trivial.

Under normal circumstances, yeah, I'm with you